ui/imodel-components-react/src/test/inputs/QuantityNumberInput.test.tsx

```diff
@@ -472,13 +472,13 @@ describe("<QuantityNumberInput />", () => {
       const wrapper = render(<QuantityNumberInput persistenceValue={value} quantityType={QuantityType.Length} step={.25} onChange={handleChange} />);
       const input = wrapper.container.querySelector("input");
       expect(input).not.to.be.null;
-      expect((input as HTMLInputElement).value).to.eq("1.");
+      expect((input as HTMLInputElement).value).to.eq("1.0");
```

Comment:
This file is moving to the iTwin/appuii repo (#5149).

Reply:
Thanks for the heads-up !
I've informed Arun and will liaise with him about the conflict when it arises.

Reply:
The fix will be to delete the file in either PR.
@NancyMcCallB @raplemie fyi test failure incoming in your repo.